encoding/thrift/thriftrw-plugin-yarpc/main.go

```diff
@@ -45,7 +45,8 @@ var (
 		"Function used to wrap generic Thrift oneway function handlers into YARPC handlers")
 	_noGomock = flag.Bool("no-gomock", false,
 		"Don't generate gomock mocks for service clients")
-	_noFx = flag.Bool("no-fx", false, "Don't generate Fx module")
+	_noFx                 = flag.Bool("no-fx", false, "Don't generate Fx module")
+	_tchannelSanitization = flag.Bool("tchannel-sanitization", false, "Enable tchannel context sanitization")
```

Comment:
I might name this `sanitize-tchannel` instead, I get why you named it this but it just reads more nicely